terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,86 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                = "company-requests" # this must match the stack name the service deploys into
```

Comment:
Is this the correct cluster? Looking at the page it isn't actually in one though accounts filing api is in filing maintain: https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/4217208873/ECS+Cluster+Architecture

Reply:
Fixed to `filing-maintain-service`